addons/mail/models/mail_user_settings.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class MailUserSettings(models.Model):
+    _name = 'mail.user.settings'
```

Comment:
``res.users.settings`` probably, as this is linked to user mainly, and used in discuss. A ``mail.user`` means nothing currently in Odoo.